src/components/StoriesGroup/StoriesGroup.tsx

```diff
@@ -119,21 +164,24 @@ export const StoriesGroup = ({
 
     // case when groups has changed and indexs has ceased to be valid
     if (currentGroup === undefined || currentItems[itemIndex] === undefined) {
+        const newIndex: StoriesGroupIndex = index;
```

Comment:
We might not have `index` here since it is optional

Reply:
`index: StoriesGroupIndex` and never `index === undefined`

Reply:
Wouldn't it be better to copy the object here?

Reply:
yes, you're right. Looks like there's a bug here. Oh, this reference js.

Reply:
[7332626](https://github.com/gravity-ui/uikit/pull/464/commits/7332626e615649a01b9f575d718fe9cf7a302f8e)

Reply:
Okay, I've forgot that there is default value